src/app/locationInfo/LocationInfo.jsx

```diff
@@ -68,14 +55,26 @@ class LocationInfo extends Component {
           title="Location Info" />
         <ProgressBar step={0} steps={routes.length} />
         <LocationHeader />
-        {routes.map((route, i) => (
-          <FieldItem
-            key={route[0]}
-            title={route[2]}
-            linkTo={`${this.props.location.pathname}/${route[0]}`}
-            updatedAt={FAKE_DATA[i]}
-          />
-        ))}
+        {
+          routes.map((route, i) => {
+            const prop1 = route[3];
```

Comment:
The more we add to the routes, the less clear it is to me why it's an array of arrays, requiring all these magic numbers to access props based on index, as opposed to an array of objects, which would allow e.g. just:
```
const { prop1, prop2 } = route;
```

Reply:
Fixed in b87dfec18c06b679b1b49a52590e15f67a00c5e5